package.json

```diff
@@ -34,11 +34,13 @@
     "node-sass": "^3.1.2"
   },
   "devDependencies": {
+    "eslint": "^1.9.0",
     "mocha": "2.2.5",
     "proxyquire": "1.6.0",
     "sinon": "1.15.4"
   },
   "scripts": {
-    "test": "NODE_ENV=test mocha"
+    "test": "NODE_ENV=test mocha",
+    "pretest": "eslint *.js src test"
```

Comment:
eslint only runs on certain file types and automatically ignores node_modules. So I think this line can be:

`"pretest": "eslint ."`